test/server/lib/DocApi.ts

```diff
@@ -4775,67 +4861,79 @@ function testDocApi() {
       describe('webhook update', function () {
 
         it('should work correctly', async function () {
-
-
           async function check(fields: any, status: number, error?: RegExp | string,
                                expectedFieldsCallback?: (fields: any) => any) {
 
-            let savedTableId = 'Table1';
             const origFields = {
               tableId: 'Table1',
               eventTypes: ['add'],
               isReadyColumn: 'B',
               name: 'My Webhook',
               memo: 'Sync store',
+              columnIds: 'A'
             };
 
             // subscribe
-            const webhook = await subscribe('foo', docId, origFields);
+            const { data } = await axios.post(
```

Comment:
Is this change just to use newer `/webhooks` instead of older `/_subscribe` endpoint?

Reply:
I think it's good to test the newer endpoint but I don't remember if in this case something went wrong with the older
Do you want me to create a specific subscribe function with the new endpoint call and rename the old one with `oldEndpointsubscribe` or something like that